src/app/components/no-tokens-found/no-tokens-found.tsx

```diff
@@ -0,0 +1,20 @@
+import { css } from 'leather-styles/css';
+import { VStack, styled } from 'leather-styles/jsx';
+
+export function NoTokensFound() {
+  return (
+    <VStack h="100%" justifyContent="center">
+      <styled.span
+        className={css({
+          textAlign: 'center',
+          padding: '20px',
+          color: 'gray',
```

Comment:
It would be better not to hard-code the color and font and to use our [tokens](https://github.com/leather-io/mono/blob/dev/packages/tokens/src/tokens.ts) or [typography](https://github.com/leather-io/mono/blob/dev/packages/tokens/src/typography.ts) if you can